frontend/src/app/components/containers/layout/side-bar.tsx

```diff
@@ -110,7 +115,12 @@ export const SideBar: FC = () => {
       {/* <!-- menu items for the organization --> */}
       <ul className="nav nav-pills flex-column mb-auto comp-nav-item-list">
         {menueItems.map((item, idx) => {
-          return renderSideBarMenuItem(idx, item);
+          if (UserService.hasRole(Roles.CEEB) && !item.roles?.includes('CEEB'))
+            {
+              // Do not display this hence return null
+              return null;
+            }
+            return renderSideBarMenuItem(idx, item);
```

Comment:
there's a better way to do this, but it does involve making a change to your roles property to an array. Instead of returning a null this way you'll want to do something like this
```
          if (!UserService.hasRole(Roles.CEEB) && !item.roles?.includes("CEEB")) {
            return renderSideBarMenuItem(idx, item);
          }
```

the map function will still return a null, you just don't need to be explicitly returning a null